pdns/lua-base4.cc

```diff
@@ -289,10 +327,11 @@ void BaseLua4::prepareContext() {
   d_lw->writeVariable("pdns", d_pd);
 }
 
-void BaseLua4::loadStream(std::istream &is) {
+void BaseLua4::loadStream(std::istream &is, bool doPostLoad) {
```

Comment:
tidy would like a longer var name here

Reply:
it feels like can of worms, but i'll give it a try.